### DISCUSSION TITLE
Header, Navbar - input 처리 방식

### DISCUSSION THREAD

**Discussion by eeseung** [prgrms-fe-devcourse/FEDC4_CUCUMIS_Pocojang#48] [General]
Header나 Navbar 위치에 input이 있는 경우가 있습니다.
- Header
  - `DeveloperListPage`
  - `ProjectListPage`
- Navbar
  - `DeveloperDetailPage`
  - `ProjectDetailPage`
  - `DMPage`

input value 처리 때문에 Header나 Navbar가 아닌 페이지 내부에서 컴포넌트를 따로 만들어야할 것 같은데 어떻게 생각하시나요? 다른 좋은 방법이 있을지 여쭤봅니다!

**Comment by kutta97** [comment]
저는 `page` 별로 `store` 를 두고, `navbar` store 와 연결할 수 있을 것 같습니다..!

**Reply by eeseung** [reply]
넵 그럼 layout store에서 처리해 보겠습니다! 🙇‍♀️
